src/app/backend/cert/ecdsa/creator.go

```diff
@@ -54,8 +55,15 @@ func (self *ecdsaCreator) GenerateCertificate(key interface{}) []byte {
 	ecdsaKey := self.getKey(key)
 	pod := self.getDashboardPod()
 
+	notBefore := time.Now()
+	validFor, _ := time.ParseDuration("8760h")
+	notAfter := notBefore.Add(validFor)
+
+
 	template := x509.Certificate{
 		SerialNumber: self.generateSerialNumber(),
+		NotAfter: notAfter,
+		NotBefore: notBefore,
```

Comment:
To pass check on travis, add spaces before `notAfter|notBefore` and align its position with `self.generateSerialNumber(),` on the previous line:
```
		SerialNumber: self.generateSerialNumber(),
		NotAfter:     notAfter,
		NotBefore:    notBefore,
```

Reply:
Executing the `gofmt -w src/app/backend/cert/ecdsa/creator.go` will do the trick